incubator/hnc/pkg/reconcilers/hierarchical_namespace.go

```diff
@@ -30,20 +33,72 @@ type HierarchicalNamespaceReconciler struct {
 }
 
 // Reconcile sets up some basic variables and then calls the business logic.
-// It currently only generates some logs for testing.
+// It currently handles basic creation of namespace and hierarchyconfig.
 func (r *HierarchicalNamespaceReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	// TODO report error state if the webhook is bypassed - see issue
 	//  https://github.com/kubernetes-sigs/multi-tenancy/issues/459
 	if ex[req.Namespace] {
 		return ctrl.Result{}, nil
 	}
 
+	ctx := context.Background()
 	log := r.Log.WithValues("trigger", req.NamespacedName)
 	log.Info("Reconciling HNS")
 
+	nm := req.Name
+	pnm := req.Namespace
```

Comment:
Does "pnm" stand for "parent name"?  IIUC, req.Name is the name of an object and req.Namespace is the namespace of an object? In this sense, "pnm" might be confusing? Feel free to correct me if I am wrong.

Reply:
Yes. The `req` here stands for the request to reconcile an hns object, of which the namespace is the parent namespace name and the name is the self-serve subnamespace name.

I will add a comment to explain this. LMK if you think other names would be better :)